src/fosslight_yocto/create_oss_report_for_yocto.py

```
@@ -683,7 +683,7 @@ def get_license_query_to_db(license_name):
         sql_query = """SELECT LM.LICENSE_NAME, LICENSE_TYPE FROM LICENSE_MASTER AS LM
         LEFT OUTER JOIN LICENSE_NICKNAME AS LN ON LM.LICENSE_NAME = LN.LICENSE_NAME
         WHERE LM.LICENSE_NAME='{license_name}' OR LM.SHORT_IDENTIFIER = '{license_name}' OR LN.LICENSE_NICKNAME = '{license_name}' """.format(
-            license_name=pymysql.escape_string(license_name))
+            license_name=pymysql.converters.escape_string(license_name))
```

Ruff (0.15.10)

[error] Possible SQL injection vector through string-based query construction

(S608)